ckanext/recombinant/i18n/fr/LC_MESSAGES/ckanext-recombinant.po

```diff
@@ -29,62 +29,62 @@ msgstr "Votre fichier a été téléchargé avec succès dans le système centra
 
 #: ckanext/recombinant/controller.py:95
 msgid "Required field"
-msgstr ""
+msgstr "Champ obligatoire"
 
 #: ckanext/recombinant/controller.py:113
 msgid "Wrong number of fields, expected {num}"
-msgstr ""
+msgstr "Nombre de champs erroné, prévu {num}"
 
 #: ckanext/recombinant/controller.py:125
 msgid "Invalid fields"
-msgstr ""
+msgstr "Champs non valides"
 
 #: ckanext/recombinant/controller.py:128
 #, python-format
 msgid "No matching records found \"%s\""
-msgstr ""
+msgstr "Aucun document correspondant n’a été trouvé"
```

Comment:
you'll need the %s in here

Reply:
Oh yeah I missed that. Just added it now.